src/BunyanDebugStream.coffee

```diff
@@ -224,8 +223,8 @@ class BunyanDebugStream extends Writable
             #{date}#{processStr}#{levelPrefix}#{src}#{prefixes}#{applyColors message, colorsToApply}
         "
 
-        line += "\n#{INDENT}#{request}" if request?
-        line += "\n" + applyColors(values.join('\n'), colorsToApply) if values.length > 0
+        line += "\n#{@_indent}#{request}" if request?
+        line += "\n" + values.map((v) -> applyColors(v, colorsToApply)).join('\n') if values.length > 0
```

Comment:
I'm using ConEmu, and it only display the first line of the parsed json object in color.
![image](https://cloud.githubusercontent.com/assets/2390257/23595289/b899986a-025b-11e7-8363-61d01bc03f1c.png)